anvil-rstudio-bioconductor/Dockerfile

```diff
@@ -1,151 +1,38 @@
-	liblapack-dev \
-	libv8-3.14-dev \
-	libgtkmm-2.4-dev \
-	libmpfr-dev \
-	libudunits2-dev \
-	libmodule-build-perl \
-	libapparmor-dev \
-	libgeos-dev \
-	libprotoc-dev \
-	librdf0-dev \
-	libmagick++-dev \
-	libpoppler-cpp-dev \
-	libprotobuf-dev \
-	libperl-dev \
-	libz-dev \
-	liblzma-dev \
-	## software - perl extentions and modules
-	libarchive-extract-perl \
-	libfile-copy-recursive-perl \
-	libcgi-pm-perl \
-	libdbi-perl \
-	libdbd-mysql-perl \
-	libxml-simple-perl \
-	## Databases and other software
-	sqlite \
-	openmpi-bin \
-	mpi-default-bin \
-	openmpi-common \
-	openmpi-doc \
-	tcl8.5-dev \
-	tk-dev \
-	openjdk-8-jdk \
-	imagemagick \
-	tabix \
-	ggobi \
-	graphviz \
-	protobuf-compiler \
-	jags \
-	## Additional resources
-	xfonts-100dpi \
-	xfonts-75dpi \
-	biber \
-	&& apt-get clean \
-	&& rm -rf /var/lib/apt/lists/*
-
-## These two libraries don't install in the above section--WHY?
-RUN apt-get update \
-	&& apt-get -y --no-install-recommends install \
-	default-jdk \
-	libjpeg-dev \
-	libjpeg-turbo8-dev \
-	libjpeg8-dev \
-	&& apt-get clean \
-	&& rm -rf /var/lib/apt/lists/*
-
-## Python installations
-RUN apt-get update \
-	&& apt-get -y --no-install-recommends install python-dev \
-	&& pip install wheel \
-	## Install sklearn and pandas on python
-	&& pip install sklearn \
-	pandas \
-	pyyaml \
-	cwltool \
-	&& apt-get clean \
-	&& rm -rf /var/lib/apt/lists/*
-
-# Install libsbml and xvfb
-RUN cd /tmp \
-	## libsbml
-	&& curl -O https://s3.amazonaws.com/linux-provisioning/libSBML-5.10.2-core-src.tar.gz \
-	&& tar zxf libSBML-5.10.2-core-src.tar.gz \
-	&& cd libsbml-5.10.2 \
-	&& ./configure --enable-layout \
-	&& make \
-	&& make install \
-	## xvfb install
-	&& cd /tmp \
-	&& curl -SL https://github.com/just-containers/s6-overlay/releases/download/v1.21.8.0/s6-overlay-amd64.tar.gz | tar -xzC / \
-	&& apt-get update && apt-get install -y --no-install-recommends xvfb \
-	&& mkdir -p /etc/services.d/xvfb/ \
-	## Clean libsbml, and tar.gz files
-	&& rm -rf /tmp/libsbml-5.10.2 \
-	&& rm -rf /tmp/libSBML-5.10.2-core-src.tar.gz \
-	## apt-get clean and remove cache
-	&& apt-get clean \
-	&& rm -rf /var/lib/apt/lists/*
-
-COPY ./deps/xvfb_init /etc/services.d/xvfb/run
-
-# Install Bioconductor packages
-RUN curl -O https://raw.githubusercontent.com/Bioconductor/anvil-docker/master/anvil-rstudio-bioconductor/install.R \
-	&& R -f install.R \
-	&& rm -rf install.R
-
-# At this point, the image has installed all of its R and pip packages so we can switch R_LIBS and PIP_USER back so that user installed packages persist
-RUN mv /usr/local/lib/R/etc/Renviron.site.temp /usr/local/lib/R/etc/Renviron.site
-# Add back other env vars
-RUN echo "TERRA_R_PLATFORM='anvil-rstudio-bioconductor'" >> /usr/local/lib/R/etc/Renviron.site \
-    && echo "TERRA_R_PLATFORM_BINARY_VERSION='0.99.1'" >> /usr/local/lib/R/etc/Renviron.site
-
-USER root
-
-# Init command for s6-overlay
-CMD ["/init"]
+FROM bioconductor/bioconductor_docker:RELEASE_3_12
```

Comment:
does this image install all of its packages as root user and not rstudio user? if so then these changes look good in terms of persistence :) I would add a comment to the top of this file like the comment in https://github.com/anvilproject/anvil-docker/blob/master/anvil-rstudio-base/Dockerfile#L3 since the base image is now deprecated

Reply:
There are no packages in that image. Just system dependencies.

Reply:
okay I just wanted to make sure that anything installed in that image is installed as root user